### PR TITLE
docs(firestore-incremental-capture): make docs more comprehensive

### PR DESCRIPTION
**Pull request by cabljac**

### CODE REVIEW

File: firestore-incremental-capture/README.md [hunk 1]
```diff
@@ -26,11 +30,33 @@ If one does not exist, you can create one with the following script:
...
 ```
 
+Note that this extension currently only works on database instances in `firestore-native` mode.
+
+### Billing
+
+To install an extension, your project must be on the Blaze (pay as you go) plan. You will be charged a small amount (typically around $0.01/month) for the Firebase resources required by this extension (even if it is not used).
+This extension uses other Firebase and Google Cloud Platform services, which have associated charges if you exceed the service's no-cost tier:
+
+- Cloud Firestore
```
Comment (cabljac) [inline firestore-incremental-capture/README.md:44]: TODO: replace with services actually used

File: firestore-incremental-capture/POSTINSTALL.md [hunk 2]
```diff
@@ -13,35 +13,41 @@ More information on this can be found [here](https://cloud.google.com/sdk/gcloud
...
-    --project=${param:PROJECT_ID} \
-    --member=serviceAccount:SA_EMAIL \
-    --role=roles/artifactregistry.writer
-  ```
+
+```bash
+  gcloud artifacts repositories add-iam-policy-binding ${param:EXT_INSTANCE_ID} \
+  --location=${param:LOCATION} \
+  --project=${param:PROJECT_ID} \
+  --member=serviceAccount:SA_EMAIL \
```
Comment (cabljac) [inline firestore-incremental-capture/POSTINSTALL.md:45]: I think we should change `SA_EMAIL` to something a bit more clear maybe

File: firestore-incremental-capture/POSTINSTALL.md [hunk 3]
```diff
@@ -13,35 +13,43 @@ More information on this can be found [here](https://cloud.google.com/sdk/gcloud
...
```
Comment (dackers86) [inline firestore-incremental-capture/POSTINSTALL.md:47]:
```suggestion
  --member=serviceAccount:SERVICE_ACCOUNT_EMAIL \
```

File: firestore-incremental-capture/PREINSTALL.md [hunk 4]
```diff
@@ -18,7 +22,28 @@ If one does not exist, you can create one with the following script:
     gcloud alpha firestore databases create \
     --database=DATABASE_ID \
     --location=LOCATION \
-    --type=DATABASE_TYPE \
+    --type=firestore-native \
     --project=PROJECT_ID
-
 ```
+
+Note that this extension currently only works on database instances in `firestore-native` mode.
```
Comment (dackers86) [inline firestore-incremental-capture/PREINSTALL.md:29]:
```suggestion
Note that this extension currently only supports database instances in `firestore-native` mode.
```

File: firestore-incremental-capture/PREINSTALL.md [hunk 5]
```diff
@@ -18,7 +22,28 @@ If one does not exist, you can create one with the following script:
...
+- Cloud EventArc
+- Cloud Functions (See [FAQs](https://firebase.google.com/support/faq#extensions-pricing))
+
+[Learn more about Firebase billing](https://firebase.google.com/pricing).
+
+### Additional Uninstall Steps
+
+> ⚠️ The extension does not delete various resources automatically on uninstall.
+
+After you have uninstalled this extension, you will probably want to remove the dataflow pipeline which was set up. You can do this through the
```
Comment (dackers86) [inline firestore-incremental-capture/PREINSTALL.md:48]:
```suggestion
After you have uninstalled this extension, you will be required to remove the dataflow pipeline which was set up. You can do this through the
```

File: firestore-incremental-capture/extension.yaml [hunk 6]
```diff
@@ -221,59 +164,45 @@ params:
     label: Collection path
     description: >
       What is the path to the collection that contains the strings that you want
-      to translate?
-    example: sync
+      to capture all changes of? Use `*` to capture all collections.
```
Comment (dackers86) [inline firestore-incremental-capture/extension.yaml:167]:
```suggestion
      to capture all changes of? Use `{document=**}` to capture all collections.
```

**Review by dackers86 — COMMENTED** [review]
Suggested updates

**Pull request merged**